controllers/admin/AdminCartsController.php

```diff
@@ -980,7 +980,6 @@ public function renderList()
         } else {
             $helper->list_skip_actions['delete'] = (array) $skip_list;
         }
-        $helper->force_show_bulk_actions = true;
```

Comment:
you remove this feature for each case? Because the issue is about "when there is no shopping carts with no ordered status".